src/zepto.js

```diff
@@ -849,7 +849,16 @@ var Zepto = (function() {
       // arguments can be nodes, arrays of nodes, Zepto objects and HTML strings
       var argType, nodes = $.map(arguments, function(arg) {
             argType = type(arg)
-            return argType == "object" || argType == "array" || arg == null ?
+            if (argType == "array"){
+              var arr = []
```

Comment:
No, you're not. All variables that are declared anywhere in a function are always declared, please see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var#var_hoisting.